src/widgets/FileWidget/index.js

```diff
@@ -781,6 +847,7 @@ FileWidget.defaultProps = {
   ProgressComponent: undefined,
   downloadable: undefined,
   downloadBasepath: undefined,
+  removableFile: false,
```

Comment:
Please use the same interface used on the `ArrayWidget` and `ScheduleWidget` (we should be as consistent as possible with https://github.com/rjsf-team/react-jsonschema-form.

Instead of creating a new prop, please use `uiSchema['ui:options'].removable`. The default should be `true`.

Reply:
@CareLuLu-Gabriel thanks for the review. I added the changes to 43916049a81a09012a1ee33754aec11927226c5e.